### PR TITLE
Resolve known image tags to digest

### LINKED ISSUE #3: Resolve all images to a digest
Images with tags can be updated at any time meaning we don't know what the actual content of the helm chart is at any given time. This also means that our tests can be foiled if an image is updated in an incompatible way.

By resolving all images in the config yaml to digests, we can lock down the exact versions. To get a new version of an image, that chart will need to be updated and deployed

### PR DESCRIPTION
**Pull request by dprotaso**
Resolves #3 - redo of #57

### CODE REVIEW

**Comment by dprotaso** [comment]
Hmm... where are the FATs checks? ie. https://github.com/projectriff/charts/pull/57/checks

**Comment by dprotaso** [comment]
nm - delayed

File: charts/istio/values.yaml [hunk 1]
```diff
@@ -1,18 +1,28 @@
 # derived from https://github.com/knative/serving/blob/master/third_party/istio-1.3.3/values-lean.yaml
+# also see https://istio.io/docs/reference/config/installation-options/
 
 global:
   proxy:
     # Enable proxy to write access log to /dev/stdout.
     accessLogFile: "/dev/stdout"
     accessLogEncoding: 'JSON'
     autoInject: disabled
+    image: docker.io/istio/proxyv2:1.3.3
```
Comment (scothis) [inline charts/istio/values.yaml]: rm, no need to redefine the default that will only make it harder to update to 1.3.4

**Comment by dprotaso** [comment]
Intent was to do the tag to digest resolution on the values for the Istio
images- that requires explicit declarations

**Pull request merged**